README.md

```diff
@@ -11,8 +11,6 @@ If user is cluster admin (`osbs_is_admin`), the following is also performed:
 - Sets up rolebindings for specified users, groups and service accounts
 
 For orchestrator namespaces (`osbs_orchestrator`):
-- reactor-config-secret is generated and stored in `osbs_generated_config_path`
-  use osbs-secret to import it
 - client-config-secret is generated and stored in `osbs_generated_config_path`
   use osbs-secret to import it
```

Comment:
Should `- client-config-secret is generated ...` also be removed?

Reply:
yes thanks, removed